### DISCUSSION TITLE
I don't understand how to do a simple heightening the contrast but keeping the same quality/file size

### DISCUSSION THREAD

**Discussion by evanescente-ondine** [ImageMagick/ImageMagick#4719] [Help]
Version: ImageMagick 7.1.0-19 Q16-HDRI x86_64 2021-12-22 https://imagemagick.org

Hi, I have a book, scanned, quite big... with 700 pages. Reducing the size isn't my purpose, I just want to add contrast, remove the color, assuming it would indeed also remove weight.
I'm frankly overwhelmed by the abundance of options I could never master. I tried :
convert -color
space gray -density 300 -quality 100  ici
1.pdf ici1_essai.pdf

You can see the difference, in the same order:
https://drive.google.com/file/d/10NuEWDBmnfbnQUK6sk_j1NYaR8XismrC/view?usp=sharing
The quality is worse, yet the size is 5 to ten times bigger (70k to 400 upt o 1500 ko, with density = 600).
Ideally what I want from correcting the same picture above with gimp, by removing the alpha channel, then changing colorspace into gray, then removing the background with "smart" automatic selection: https://drive.google.com/file/d/1n-olzttPoceugeTpytr3CZJtxHgXE7LR/view?usp=sharing
I don't understand how simply taking the same pixels but removing the colors and heightening intensity for more contrast can possibly result in a bigger file size.

**Comment by snibgo** [comment]
You show screenshots but don't link to the pdf files, so we can only guess. I guess that your input pdf has image compression, and your output doesn't.

**Comment by evanescente-ondine** [comment]
sorry.
Here's the isolated page (p90-91 in the book) and the original book (30 Mo, not that big actually... but so slow to read, it must be the compression, you're right):
https://drive.google.com/file/d/1K1xaEgAjsNtfOgUZ5sr8pjaycJgxcSak/view?usp=sharing, https://drive.google.com/file/d/1rdReTYrvUAG3rXV3rWrck55Bh7gLz7ev/view?usp=sharing

**Comment by evanescente-ondine** [comment]
can I have a hint how to keep the same file but increase the contrast a little or merely switch the background to white ?

**Reply by fmw42** [reply]
See -brightness-contrast for general brightness and/or contrast changes.
